addons/web/static/src/js/fields/field_registry.js

```diff
@@ -41,6 +41,7 @@ registry
     .add('attachment_image', basic_fields.AttachmentImage)
     .add('image_selection', basic_fields.ImageSelection)
     .add('label_selection', basic_fields.LabelSelection)
+    .add('kanban_label_selection', basic_fields.LabelSelection)
```

Comment:
i'd add a `// deprecated, please use 'label_selection' instead` comment
(and i'd do the same for state_selection)

Reply:
done